internal/boot/context-managers.go

```diff
@@ -111,7 +111,7 @@ func WithBootedQemuImage(image string, ns NetNS, f func() error) error {
 		}
 
 		var qemuCmd *exec.Cmd
-		if common.CurrentArch() == arch.ARCH_X86_64.String() {
+		if arch.CurrentArch() == arch.ARCH_X86_64 {
```

Comment:
Could also add something like `arch.IsX86()` (and `arch.IsAarch64` etc...)

Reply:
yeah that'd be nice.  Also should we rename it to `arch.Current()` (or `arch.GetCurrent()`) to avoid repetition of the word `arch`?

Reply:
done! reads much better